forui/lib/src/widgets/progress.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:meta/meta.dart';
+
+import 'package:forui/forui.dart';
+
+/// A widget that shows progress on a line.
```

Comment:
I think the docs here are lacking. Refer to the other widgets, it includes a `See:` portion as well.